client/src/components/DashboardHeader/index.jsx

```diff
@@ -40,27 +40,27 @@ const DashboardHeader = ({
         </li>
         <li>
           <Link to={`${baseUrl}?tab=repositories`} className={getClassName(tabs.repositories, activeTab)}>
-            Repositories{repoCount && <span>{repoCount}</span>}
+            Repositories{<span>{repoCount || 0}</span>}
```

Comment:
Could we avoid using curly braces around <span> element?

Reply:
Fixed